ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsFetch.scala

```diff
@@ -406,6 +407,16 @@ private[http] object ContractsFetch {
     }.void
   }
 
+  private def conflation[D, C: InsertDeleteStep.Cid]
+      : Flow[InsertDeleteStep[D, C], InsertDeleteStep[D, C], NotUsed] = {
+    // when considering this cost, keep in mind that each deleteContracts
+    // may entail a table scan.  Backpressure indicates that DB operations
+    // are slow, the idea here is to set the DB up for success
+    val maxCost = 250L
```

Comment:
Should this be configurable, rather than hard-coded?

Reply:
No, I don't think so. The benefit comes from setting a [reasonably large] limit, more than what that limit is.

Reply:
@S11001001 how did we come up with this `maxCost` value , also can you remind me of the semantics of delete and why it may entail a table scan ?

Reply:
We came up with it by me taking the number 200 from the client output and adding 50 to it. It's a buffer size; it doesn’t really matter what the number is so much as that there is a number and it is reasonably "large" so that the buffer doesn't grow without bound but still gives you good-sized "chunks".

As for the deletes, it depends on how the DB implements `delete from contracts where contract_id in (id1, id2, ...)`.